amplify/backend/api/collectionarchives/override.ts

```diff
@@ -0,0 +1,11 @@
+import { AmplifyApiGraphQlResourceStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+  resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig = {
+    ...resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig,
+    instanceCount: 3
```

Comment:
For dev, please change to 1; 3 is for production use.
Is the file automatically created?

Reply:
We added this file. I think it would be best to leave it at 3 in the repository and manually change it when creating a dev instance. That way there will be less steps for the systems team to have to complete when doing deployments.